kinto_elasticsearch/command_reindex.py

```diff
@@ -0,0 +1,119 @@
+import argparse
+import sys
+
+from pyramid.paster import bootstrap
+
+from kinto.core.storage.exceptions import RecordNotFoundError
+from kinto.core.storage import Sort, Filter
+from kinto.core.utils import COMPARISON
+
+
+DEFAULT_CONFIG_FILE = 'config/kinto.ini'
+
+
+def main(cli_args=None):
+    if cli_args is None:
+        cli_args = sys.argv[1:]
+
+    parser = argparse.ArgumentParser()
```

Comment:
Do you think we could work on adding an entry-point so that plugins can add sub-commands to the kinto command?

Reply:
I have created https://github.com/Kinto/kinto/issues/1330 about that.